config/rbac/role.yaml

```diff
@@ -32,3 +32,11 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
It shall be generated by `controller-gen` instead of writing manually.

Reply:
See https://github.com/kubesphere-sigs/ks-releaser/pull/14